tests/test_yaml.py

```diff
@@ -61,13 +63,15 @@ def test_invalid_line():
 
     assert config.as_text() == """line_a: True
 line_b: !2
-        ^
---------
-Basic YAML parsing error
-- could not determine a constructor for the tag '!2'
---------
+#       ^
+# --------
+# Basic YAML parsing error
+# - could not determine a constructor for the tag '!2'
+# --------
 line_c: 8"""
 
+    assert isinstance(config.as_text(), unicode)
```

Comment:
Well that's annoying..Travis has an alert up that says build logs are currently unavailable. I'm guessing that the issue (since all python 3 builds failed) is that in python 3 this is `str`.